.travis.yml

```diff
@@ -0,0 +1,20 @@
+
+language: cpp
+
+compiler:
+  - clang
+  - gcc
+
+before_install:
+  - sudo apt-get update
+
+install:
+  - sudo apt-get install -y build-essential cmake git python-dev libboost-python-dev python-numpy libeigen3-dev
```

Comment:
@paulinus, since we can build without boost, I think than now we can remove libboost-python-dev  from the required package.